frontend/src/pages/DataCatalog/MethodologyTab.tsx

```diff
@@ -716,16 +716,44 @@ function MethodologyTab() {
                     race/ethnicity.
                   </li>
                   <li>
-                    <b>Asian (Non-Hispanic)</b>: A single race (Asian), not
-                    Hispanic/Latino.
+                    <b>American Indian and Alaska Native (Non-Hispanic)</b>: A
```

Comment:
For each of these we should either remove "(Non-Hispanic)" and then have a single entry at the bottom that explains NH races, or simply include in each entry that this NH race group excludes anyone who identifies Hispanic as their ethnicity